kubernetes/launch-job.sh

```diff
@@ -227,7 +226,9 @@ if [ $EXTRA_VICTIMPLAY_GPUS -gt 0 ]; then
       "$CPP_IMAGE" \
       $VOLUME_FLAGS \
       --command "$VICTIMPLAY_CMD" \
+      --restart-on-failure \
+      --memory 72Gi \
```

Comment:
i don't actually know if memory spikes this high but I saw one of my match jobs get `OOMKilled` with `--memory 64Gi` 🤷 I'll rerun that job and track the memory usage to see if I should change this number